lib/money/money.rb

```diff
@@ -213,12 +218,8 @@ def to_liquid
     cents
   end
 
-  def to_json(options = {})
-    to_s
-  end
-
-  def as_json(*args)
-    to_s
+  def as_json(*_args)
+    { value: to_s(:amount), currency: currency.to_s }
```

Comment:
Should we prefer `currency.iso_code` here instead to be a bit more clear and future proof if we want to_s to be more human readable?